ci/test/04_install.sh

```diff
@@ -87,13 +87,20 @@ fi
 DOCKER_EXEC echo "Free disk space:"
 DOCKER_EXEC df -h
 
-if [ "$RUN_FUZZ_TESTS" = "true" ] || [ "$RUN_UNIT_TESTS" = "true" ] || [ "$RUN_UNIT_TESTS_SEQUENTIAL" = "true" ]; then
+if [ "$RUN_FUZZ_TESTS" = "true" ]; then
   if [ ! -d "${DIR_QA_ASSETS}" ]; then
     DOCKER_EXEC git clone --depth=1 https://github.com/bitcoin-core/qa-assets "${DIR_QA_ASSETS}"
   fi
 
   export DIR_FUZZ_IN=${DIR_QA_ASSETS}/fuzz_seed_corpus/
   export DIR_UNIT_TEST_DATA=${DIR_QA_ASSETS}/unit_test_data/
```

Comment:
Is this still needed?

Reply:
Yes, in case both `"$RUN_FUZZ_TESTS" = "true"` and `"$RUN_UNIT_TESTS" = "true"`.

Reply:
It is not possible to run both.

Fuzz tests can only run with libfuzzer (right now), in which case unit tests don't work and vice-versa

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/23938#issuecomment-1006633606).